notebooks/GabeDemo.ipynb

```diff
@@ -276,14 +285,14 @@
      "text": [
       "Model is valid? False\n",
       "Reasons why:\n",
-      " -Graph did not have 1 instances of https://brickschema.org/schema/Brick#AHU\n",
-      " -Graph did not have 2 instances of urn:ashrae/g36/4.2/vav-with-reheat/vav-with-reheat\n"
+      " -Graph did not have 2 instances of urn:ashrae/g36/4.2/vav-with-reheat/vav-with-reheat\n",
+      " -Graph did not have 1 instances of https://brickschema.org/schema/Brick#AHU\n"
      ]
     }
    ],
    "source": [
     "# pass our collections of shapes in to validate the model\n",
-    "ctx = model.validate([manifest.get_shape_collection(), constraints.get_shape_collection(), \n",
+    "ctx = model.validate([model.get_shape_collection(), constraints.get_shape_collection(), \n",
```

Comment:
@TShapinsky @gtfierro Should a models manifest automatically be included in it's validation?

Reply:
If no arguments are provided to `validate()`, then it should default to the manifest; otherwise, just use the list of shape collections provided in the call to `validate`

Reply:
I've implemented this in #236